### PR TITLE
New Crowdin translations

### PR DESCRIPTION
**Pull request by dktapps**
Enjoy!

### CODE REVIEW

**Comment by dktapps** [comment]
sorry travis... blame Crowdin for 70 consecutive builds, not me... it was a bot 😂

File: src/pocketmine/lang/locale/en-PT.ini [hunk 1]
```diff
@@ -0,0 +1,159 @@
+language.name=Pirate English
```
Comment (Sandertv) [inline src/pocketmine/lang/locale/en-PT.ini]: We got a PIRATE English? 😆

Comment (dktapps) [inline src/pocketmine/lang/locale/en-PT.ini]: blame Crowdin...

**Comment by dktapps** [comment]
Ignore the failed builds, I cancelled them all.

**Comment by Muqsit** [comment]
damn @dktapps, how many languages do u know :3

File: src/pocketmine/lang/locale/en-PT.ini (hunk 1 above)
Comment (madcode15) [inline src/pocketmine/lang/locale/en-PT.ini]: I'm actual dying laughing at pirate language. 😂

Comment (dschwartz783) [inline src/pocketmine/lang/locale/en-PT.ini]: Obviously. Pirate is probably the only relevant language, matey...

**Pull request merged**